src/main/java/com/scaleunlimited/flinkcrawler/functions/BaseProcessFunction.java

```diff
@@ -11,23 +11,25 @@
 public abstract class BaseProcessFunction<IN, OUT> extends ProcessFunction<IN, OUT> {
 
     protected transient int _parallelism;
-    protected transient int _partition;
-
+    protected transient int _operatorIndex;
```

Comment:
I could change this back to _partition if you like, but I was just trying to play along with the names used by the streaming test harness.

Reply:
I think `_operatorIndex` makes more sense, but then we should use that everywhere else.